src/components/Table/GridTable.tsx

```diff
@@ -93,6 +93,8 @@ export interface GridStyle {
   levels?: Record<number, { cellCss?: Properties; firstContentColumn?: Properties }>;
   /** Allows for customization of the background color used to denote an "active" row */
   activeBgColor?: Palette;
+  /** Enables cells Highlight and hover */
+  cellHighlight?: boolean;
```

Comment:
**request:** I'm thinking I would prefer this to be an option on the `GridStyleDef` in `styles.tsx`, and not here. If that is set, then we can add in hover styles for the `cellCss` defined in the `memoizedTableStyles` function. 
The reasoning for this request is because we want to move away from people using `GridStyle` and instead choose a `GridStyleDef`. This would ensure better consistency between all of our tables.